packages/jest-snapshot/src/__tests__/snapshot.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+jest
+  .unmock('react');
+
+const fs = require('fs');
+const path = require('path');
+let React;
+
+describe('snapshot', () => {
+
+  beforeEach(() => {
+    React = require('react');
+  });
+
+  it('works with plain objects', () => {
+    const test = {
+      a: 1,
+      b: '2',
+      c: 'three',
+    };
+    expect(test).toMatchSnapshot();
+
+    // check if the file has been created correctly
+    const expectedPath = path.resolve(
+      __dirname,
+      './__snapshots__/',
+      path.basename(__filename) + '.snap'
+    );
+    let ok = true;
+    try {
+      fs.accessSync(expectedPath, fs.F_OK);
+    } catch (e) {
+      ok = false;
+    }
+    expect(ok).toBe(true);
+
+    // check if the snapshot has the correct key
+    const snapshot = JSON.parse('' + fs.readFileSync(expectedPath));
+    const prettyPrinted = (
+      '{\n\u001b[90m|\u001b[39m a: 1,\n\u001b[90m|\u001b' +
```

Comment:
Why is the data in the snapshot pretty-printed? It should definitely not store pretty printed data with colors. It may pretty-print JSON with newlines/spaces to make it readable, but not write things that come from Jest.